CMakeLists.txt

```diff
@@ -93,7 +98,7 @@ install(DIRECTORY include
 install(FILES README.md DESTINATION ${INSTALL_MISC_DIR})
 install(FILES cmake/modules/FindLUG.cmake DESTINATION ${INSTALL_MISC_DIR}/cmake/modules)
 
-# Unit test
+# unit test
```

Comment:
It's like that in all the commentaries of the file (without uppercase at the begin of the sentence)